packages/nuxt/src/module.ts

```diff
@@ -45,24 +46,16 @@ export default defineNuxtModule<ModuleOptions>({
       return
     }
 
-    // avoid unwanted behavior with different package managers
-    const schemaOrgPath = dirname(await resolvePath(Pkg))
-
     // if ssr is disabled we need to inject the client
     if (!nuxt.options.ssr)
       moduleOptions.client = true
     // enable client in dev mode
     if (typeof moduleOptions.client === 'undefined')
       moduleOptions.client = !!nuxt.options.dev
 
-    const providerPath = await resolvePath(`${schemaOrgPath}/providers/${moduleOptions.full ? 'full' : 'simple'}`)
-    const runtimePath = await resolvePath(`${schemaOrgPath}/runtime`)
-    const runtimeMockPath = await resolvePath(`${schemaOrgPath}/runtime-mock`)
-    // // set the alias for the types
-    nuxt.options.alias[AliasProvider] = providerPath
-    nuxt.options.alias[AliasRuntime] = runtimePath
-    // NOTE: ALIASING PKG IN VITE BREAKS IT, not sure why this works
```

Comment:
logic change here, we're now allowing the vite plugin to handle package resolution